backend/uclapi/roombookings/helpers.py

```diff
@@ -151,13 +175,18 @@ def _parse_datetime(start_time, end_time, search_date):
 def _serialize_rooms(room_set):
     rooms = []
     for room in room_set:
+        if room.roomclass in ROOM_TYPE_MAP:
+            classification_str = ROOM_TYPE_MAP[room.roomclass]
+        else:
+            classification_str = "Unknown Room Type"
         room_to_add = {
             "roomname": room.roomname,
             "roomid": room.roomid,
             "siteid": room.siteid,
             "sitename": room.sitename,
             "capacity": room.capacity,
             "classification": room.roomclass,
+            "classification_str": classification_str,
```

Comment:
Any chance we can give this a better name? Maybe even replace "classification" with this?

Reply:
I based this off the [Twitter API](https://developer.twitter.com/en/docs/basics/twitter-ids), so as to avoid making a breaking change. The other option would be to leave in the short code if we don't have the classification in the map? What's best in your opinion?

Reply:
Yeah not making a breaking change is important. I'm not sure what the right solution is here :/ but having _str as part of the variable name _feels_ wrong.

Reply:
How about `classification_name`?

Reply:
Yeah, definitely prefer that!